app/models/backend_api.rb

```diff
@@ -30,6 +30,8 @@ class BackendApi < ApplicationRecord
 
   has_system_name(uniqueness_scope: [:account_id])
 
+  scope :orphans, -> { joining { services.outer }.where{ BabySqueel[:backend_api_configs].backend_api_id == nil } }
```

Comment:
```suggestion
  scope :orphans, -> { joining { services.outer }.where { BabySqueel[:backend_api_configs].backend_api_id == nil } }
```
For this:
![image](https://user-images.githubusercontent.com/11318903/65066998-d2154d80-d985-11e9-938d-688120d4d3c5.png)